src/site/_includes/homepage-next.njk

```diff
@@ -23,82 +23,35 @@ externalScripts:
+            {% elif featuredCard.background %}
+            {% Img src="image/SZHNhsfjU9RbCestTGZU6N7JEWs1/8JNKYwBSBG5CtMo2KxkB.png", alt=featuredCard.alt, width="800", height="434", class="feature-card__background" %}
             {% endif %}
           </a>
+
           {% include "partials/picked-case-study.njk" %}
         </div>
       </div>
     </div>
   </header>
   {# Hero ends #}
 
-  {# Promo panel starts #}
-  {# {% if promoPanel %}
-    <article class="hero" data-type="promo">
-      <div class="wrapper">
-        <div class="hero__columns switcher" data-direction="rev">
-          <div>
-            <span aria-hidden="true" class="hero__eyebrow">{{ promoPanel.eyebrow }}</span>
-            <div class="hero__content flow">
-              <h2 class="hero__title">{{ promoPanel.title }}</h2>
-              <p>{{ promoPanel.summary }}</p>
-              <div class="cluster gutter-base flow-space-size-2">
-                <a class="button" data-type="primary" href="{{ promoPanel.primaryButtonUrl }}">{{ promoPanel.primaryButtonText }}</a>
-                <a class="button" data-type="secondary" href="{{ promoPanel.secondaryButtonUrl }}">{{ promoPanel.secondaryButtonText }}</a>
-              </div>
-            </div>
-          </div>
-          <div>
-            <img fetchpriority="high" src="{{ promoPanel.image }}" alt="" aria-hidden="true" class="hero__decor" width="{{ promoPanel.imageWidth }}" height="{{ promoPanel.imageHeight }}" />
-          </div>
-        </div>
-      </div>
-    </article>
-  {% endif %} #}
-  {# Promo panel ends #}
-
-  {# Optional divider between promo (if both enabled) #}
-  {# <hr /> #}
-
-  {# Courses start #}
-  <div class="homepage__section">
-    <div class="wrapper">
-      <div class="headline all-center flow">
-        <h2 class="headline__title">{{ courses.title }}</h2>
-        <p>{{ courses.summary }}</p>
-        <div class="headline__action">
-          <a href="/learn">{{ 'i18n.common.see_all' | i18n(locale) }}</a>
-        </div>
-      </div>
-      <div class="gap-top-size-2">
-        {% include "partials/course-cards-next.njk" %}
-      </div>
-    </div>
-  </div>
-  {# Courses end #}
-
-  {# Case studies start #}
+  {# Web Platform News start #}
   <div class="homepage__section bg-mid-bg">
     <div class="wrapper">
       <div class="headline all-center flow">
-        <h2 class="headline__title">{{ caseStudies.title }}</h2>
-        <p>{{ caseStudies.summary }}</p>
-        <div class="headline__action">
-          <a href="/tags/case-study/">{{ 'i18n.common.see_all' | i18n(locale) }}</a>
-        </div>
+        <h2 class="headline__title">{{ news.title }}</h2>
+        <p>{{ news.subTitle }}</p>
       </div>
-      <div class="homepage__case-studies switcher gap-top-size-2">
+      <div class="homepage__case-studies gap-top-size-2">
```

Comment:
These cards should stack on smaller screens, otherwise they are too narrow:

<img width="253" alt="Screenshot 2022-12-21 at 14 13 22" src="https://user-images.githubusercontent.com/1914261/208914298-eb8be748-4ed5-4096-b263-f49aeafa805a.png">

Probably "switcher" would give you that.